app/main.py

```diff
@@ -1,7 +1,25 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+
+def get_weather(api_key: str) -> None:
+    city = "Paris"
+    base_url = f"http://api.weatherapi.com/v1/current.json?key=" f"{api_key}&q={city}"
+    response = requests.get(base_url)
+    weather = response.json()
+
+    if response.status_code == 200:
+        print(
+            f"{weather["location"]["name"]}/"
+            f"{weather["location"]["country"]} "
+            f"{weather["location"]["localtime"]} "
+            f"Weather: {weather["current"]["temp_c"]} "
+            f"Celsius, {weather["current"]["condition"]["text"]}"
+        )
+    else:
+        "There is no data about weather"
```

Comment:
Bug: The string 'There is no data about weather' is not printed or returned. If the response status code is not 200, this message will not be shown. Consider using a print statement or raising an exception in the else block.